smart-contracts/contracts/cl-vault/src/contract.rs

```diff
@@ -140,6 +163,8 @@ pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractEr
         Replies::RangeIterationCreatePosition => {
             handle_iteration_create_position_reply(deps, env, msg.result)
         }
+        Replies::RangeAddToPosition => handle_add_to_position_reply(deps, env, msg.result),
```

Comment:
for naming consistency : 
handle_add_to_position_reply  -> handle_range_add_to_position_reply

Reply:
renamed in 6506fbf7a0b6aa806ae10f36c511b2961490ab25